workshops/04-Gem-Packaging-Workshop/README.md

```diff
@@ -0,0 +1,81 @@
+# Gem Packaging
+
+In Ruby we use Gems to distribute reusable components.
+
+For example, `rspec` is distributed as a Gem.
```

Comment:
More examples of gems might be useful, as I feel the phrase "reusable components" may be difficult to understand? What's a component, why would I reuse it?

Reply:
Maybe linking to their git repos too? Though i'd pick some smol simple gems for that